packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -2456,6 +2420,202 @@ contract OPContractsManager is ISemver {
         return opcmDeployer.isDevFeatureEnabled(_feature);
     }
 
+    /// @notice Helper that converts the legacy DeployInput into the new FullConfig.
+    /// @param _input The legacy DeployInput.
+    /// @param _superchainConfig The SuperchainConfig contract.
+    /// @return The new FullConfig.
+    function _toFullConfig(
+        DeployInput memory _input,
+        ISuperchainConfig _superchainConfig
```

Comment:
do we need this arg or can we just read it from storage (which is what happens in the one place that this function is called).